packages/npm-mongo/package.js

```diff
@@ -17,4 +17,5 @@ Package.onUse(function (api) {
     "NpmModuleMongodb",
     "NpmModuleMongodbVersion",
   ], "server");
+  api.addAssets('npm-mongo.d.ts', 'server');
```

Comment:
I'd rather call it `index.d.ts`, just for the convention. _(We're actually discussing it [on Slack](https://meteor-community.slack.com/archives/CS46CNDN0/p1675929707227739?thread_ts=1673006401.030039&cid=CS46CNDN0) too.)_